docs/deployment/docker.rst

```diff
@@ -16,8 +16,7 @@ restarded, the stack won't start automatically.
 
 The current strategy is:
 
- - Use a virtual machine with a well patched OS (debian, ubuntu, RHEL), djangopackages.org is using
- ubuntu 14.04
+ - Use a virtual machine with a well patched OS (debian, ubuntu, RHEL), djangopackages.org is using ubuntu 14.04
```

Comment:
@jefftriplett Is the server still using Ubuntu 14.04?

Reply:
It's still on the OG sever from ~8 or ~9 years ago AFAICT. I'd like to move it over to a new slice over one of our holiday breaks over the next month or so. It shouldn't be too bad since I moved the DB out, and it's mostly container based outside of cron-jobs. I'll open a ticket fo this.